functional-tests/wdio.conf.ts

```diff
@@ -41,10 +43,12 @@ export const config: WebdriverIO.Config = {
 	cucumberOpts: {
 		require: [
 			"./steps/**/*.ts",
-			"./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",
+			"./steps/given.ts",
+			"./support/pagePaths.ts",
+			// "./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",
```

Comment:
Should we remove this completley? No need for it anymore now we have removed wdio-cucumber-steps

Reply:
I am referring to line 48 - // "./node_modules/@nice-digital/wdio-cucumber-steps/lib/index.js",

Reply:
yes will clean that up Martin